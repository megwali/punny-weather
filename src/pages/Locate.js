import Dashboard from '../components/Dashboard';

const Locate = ({ data }) => (
  <Dashboard data={data} loadingMessage={'Doing magic... ðŸŽ©ðŸ•´'} />
);

export default Locate;
