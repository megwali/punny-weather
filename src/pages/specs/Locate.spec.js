import { create } from 'react-test-renderer';
import Locate from '../Locate';


describe('Locate', () => {
  it ('renders', () => {
    const data = {};
    const component = create(<Locate data={data} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
