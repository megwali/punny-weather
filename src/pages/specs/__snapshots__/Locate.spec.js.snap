// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locate renders 1`] = `
<div
  className="sc-hKFxyN cSGhCN"
/>
`;
